### ISSUE TITLE
Topics indented incorrectly

### ISSUE THREAD

**Issue by t-schroeder** [moodleuulm/moodle-local_boostnavigation#35]
When you

1. have "local_boostnavigation | insertcoursesectionscoursenode" enabled in Site administration,
2. have a course in topics format and
3. have course layout set to "Show one section per page" in the course settings

only the first section is indented in the nav drawer. The remaining sections are not indented.

![image](https://user-images.githubusercontent.com/24255259/57081314-bcfc9280-6cf4-11e9-815e-6875a0f06cd5.png)

P.S.: When you go to another section the indentations don't change.
P.P.S.: Collapsing the parent node works, though. It collapses all child nodes, even the ones that are not indented.

**Comment by abias** [comment]
Hi Tim,

Looking at your screenshot, I am wondering which Moodle version you are on. Since Moodle 3.5, there are icons in front of each navigation node and we have adopted these icons to local_boostnavigation as well.

So, on Moodle 3.6 or 3.7, this should look like this:
![Course__Demokurs](https://user-images.githubusercontent.com/1092118/59421065-48764280-8dce-11e9-91d8-821dcb6c4218.png)

So, if you are running an older version of Moodle and the plugin, you might have encountered a CSS bug which we didn't spot back then but which also will not be fixes anymore by us (see https://github.com/moodleuulm/moodle-local_boostnavigation#moodle-release-support).

Otherwise, if you are running Moodle 3.6 or 3.7, I am wondering why you don't have the icons in front of the section nodes.

Can you please shed some light on this?

Thanks,
Alex

**Comment by t-schroeder** [comment]
Hi Alex,

I just checked again and found that the problem is in our own theme. We copied the `#nav-drawer .list-group-item[data-key="localboostnavigationcoursesections"][data-collapse="0"] + .list-group-item[href$="#section-0"]` selector and added padding to it which of course only effects the first section. It didn't have anything to do with the icons.

Sorry for not checking this well enough before I created the issue.

Regards,
Tim
